modules/apps/transfer/types/trace_test.go

```diff
@@ -118,26 +121,25 @@ func TestValidatePrefixedDenom(t *testing.T) {
 		denom    string
 		expError bool
 	}{
-		{"prefixed denom", "transfer/channelToA/uatom", false},
-		{"prefixed denom with '/'", "transfer/channelToA/gamm/pool/1", false},
+		{"prefixed denom", "transfer/channel-1/uatom", false},
+		{"prefixed denom with '/'", "transfer/channel-1/gamm/pool/1", false},
 		{"empty prefix", "/uatom", false},
 		{"empty identifiers", "//uatom", false},
 		{"base denom", "uatom", false},
 		{"base denom with single '/'", "erc20/0x85bcBCd7e79Ec36f4fBBDc54F90C643d921151AA", false},
 		{"base denom with multiple '/'s", "gamm/pool/1", false},
-		{"invalid port ID", "(transfer)/channelToA/uatom", false},
+		{"invalid port ID", "(transfer)/channel-1/uatom", true},
 		{"empty denom", "", true},
 		{"single trace identifier", "transfer/", true},
-		{"invalid channel ID", "transfer/(channelToA)/uatom", true},
```

Comment:
had to remove this test, the `ValidatePrefixedDenom` relies on the parsing function. As the documentation indicates, an invalid channel identifier will result in the channel identifier being treated as a base denomination, thus making the test case "invalid channel id" always pass (we don't do validation on the base denomination, since it isn't ours to validate)